include/openssl/bn.h

```diff
@@ -308,6 +308,11 @@ int BN_mod_exp2_mont(BIGNUM *r, const BIGNUM *a1, const BIGNUM *p1,
                      BN_CTX *ctx, BN_MONT_CTX *m_ctx);
 int BN_mod_exp_simple(BIGNUM *r, const BIGNUM *a, const BIGNUM *p,
                       const BIGNUM *m, BN_CTX *ctx);
+int BN_mod_exp_mont_consttime_x2(BIGNUM *rr1, const BIGNUM *a1, const BIGNUM *p1,
+                                 const BIGNUM *m1, BN_MONT_CTX *in_mont1,
+                                 BIGNUM *rr2, const BIGNUM *a2, const BIGNUM *p2,
+                                 const BIGNUM *m2, BN_MONT_CTX *in_mont2,
+                                 BN_CTX *ctx);
```

Comment:
This should probably not be a public function.

Reply:
It may be so, I'm still not sure if it is worth to make it public. I put it here by analogue with other functions of similar nature like `BN_mod_exp_mont_consttime()`. Actually the implementation calls the latter function in case if CPU is not capable of this optimization, so it is like a wrapper.

Reply:
I guess it can be useful for applications that also want to use it. But in that case, I want to see real documentation for the new function instead of adding it to the list of undocumented functions. I suggest you make it an internal only function for now.

Reply:
This is fair, but probably in that case it is worth to add documentation for all existing `BN_mod_exp_mont` public functions family, as the doc just for this new function will look incomplete without others, but this task deserves separate pull-request. At the same time keeping this function separate from others related (in internal header) look inconsistent, don't it?

Reply:
All functions need to be documented. For public functions that's in a .pod file. There might be some exceptions, but I see no reason why this function should not be documented if we make it public. We're currently in a state where not all public functions are documented, and we don't want to make it worse. Since this is a new function, now is the perfect time to documented it. If you want to document some other functions that are currently missing documentation, that's great. But that they currently aren't documented is not a reason to move the documentation for this new function to a separate PR.

Reply:
I'm not sure what you mean with inconsistent. The question is if we want to make it public or not. Is it useful for other applications? Or will it only ever be used internally by OpenSSL? Currently it seems to me that it most likely will only be used by OpenSSL. But if someone does want to use it, we can make it public at that time.

Reply:
Ok, let's have it in this PR. I'll add some words for several other functions as well, so that the new function won't be sitting out of context in the pod file.